x/blob/client/testutil/integration_test.go

```diff
@@ -87,6 +88,21 @@ func (s *IntegrationTestSuite) TestSubmitPayForBlob() {
 			expectedCode: 0,
 			respType:     &sdk.TxResponse{},
 		},
+		{
+			name: "invalid share version",
+			args: []string{
+				hexNamespace,
+				hexBlob,
+				fmt.Sprintf("--from=%s", username),
+				fmt.Sprintf("--%s=%s", flags.FlagBroadcastMode, flags.BroadcastBlock),
+				fmt.Sprintf("--%s=%s", flags.FlagFees, sdk.NewCoins(sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(2))).String()),
+				fmt.Sprintf("--%s=true", flags.FlagSkipConfirmation),
+				fmt.Sprintf("--%s=1", cli.FlagShareVersion),
+			},
+			expectErr:    true,
```

Comment:
nice

Reply:
do you mean in `integration_test.go` file or create another file for unit test ?

Reply:
nvm, I think this is more than fine now that we can remove any additional validation :slightly_smiling_face: